modules/las/src/las-loader.ts

```diff
@@ -9,10 +9,28 @@ export type LASLoaderOptions = LoaderOptions & {
   las?: {
     fp64?: boolean;
     skip?: number;
-    colorDepth?: number;
+    colorDepth?: number | string;
   };
 };
 
+/**
```

Comment:
Putting it here makes files import each other circularly. We already do this for the options but let's not make it worse. Keep it in a `./types` field.